adapters/to_sql.py

```diff
@@ -45,14 +46,14 @@ def to_sql_release(dag_id, execution_date):
   # construct query
   sql_query = 'SELECT dag_id, execution_date FROM dag_run'
   sql_query += ' WHERE dag_id = "' + dag_id + '"'
-  sql_query += ' AND execution_date = "' + str(execution_date) + '"'
+  sql_query += ' AND execution_date LIKE "%' + str(execution_date) + '%"'
```

Comment:
Why are we changing that ?

Reply:
I noticed that there were some releases, most of which were triggered manually, that were not showing up on the dashboard. I figured out that this is because the execution dates contained nanoseconds, which was not originally compensated for. By changing it to like, it ensures that the execution dates can contain nanoseconds, but do not have to. The fix actually caused about 35 more releases to be available on the dashboard.

Reply:
ok. Could you please briefly document that in the code?

Reply:
Done